core/src/com/po/fuck/model/Room.java

```diff
@@ -53,13 +50,13 @@ protected void spawnEnemies() {
 
         // top left
         Manager.create(new BasicEnemy(new GeometryData(center.cpy().mulAdd(offset, -0.5f),
-                                    new Sprite(new Texture("player2.png")).getWidth(),
-                                    new Sprite(new Texture("player2.png")).getHeight(),0)));
+                SpriteInfo.getBasicAssetInfo(BasicEnemy.class).getWidth(),
+                SpriteInfo.getBasicAssetInfo(BasicEnemy.class).getHeight(),0)));
```

Comment:
Move `SpriteInfo.getBasicAssetInfo(BasicEnemy.class)` into a variable.

Reply:
Why are we passing `geometryData` into `BasicEnemy`, it should already have info about its size.